openassessment/xblock/rubric_reuse_mixin.py

```diff
@@ -0,0 +1,113 @@
+""" Mixin for functionality around the reuse of rubrics between ORAs """
+import logging
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import UsageKey
+from xblock.core import XBlock
+
+logger = logging.getLogger(__name__)
+
+
+class RubricReuseMixin:
+
+    def _get_course_ora_blocks(self):
+        """
+        Get all ORA blocks from the course.
+        Both draft and published versions are queried, with preference to draft versions
+        """
+        if hasattr(self.runtime, 'modulestore'):
+            return self.runtime.modulestore.get_items(
+                self.location.course_key,
+                qualifiers={'category': 'openassessment'},
+            )
+        else:
+            logger.info("_get_course_ora_blocks was called for {block_id} but no modulestore was available".format(
```

Comment:
There are other places where we kinda expect this and have non-error behavior? 

```python
     def is_released(self, step=None):
        """
         ....
        """
        # By default, assume that we're published, in case the runtime doesn't support publish date.
        if hasattr(self.runtime, 'modulestore'):
            is_published = self.runtime.modulestore.has_published_version(self)
        else:
            is_published = True
```